src/modules/navigator/mission.h

```diff
@@ -251,6 +254,9 @@ class Mission : public MissionBlock, public ModuleParams
 	// track location of planned mission landing
 	bool	_land_start_available{false};
 	uint16_t _land_start_index{UINT16_MAX};		/**< index of DO_LAND_START, INVALID_DO_LAND_START if no planned landing */
+	float _landing_lat{0.0f};
```

Comment:
lat/lon should be stored as double:
```suggestion
	double _landing_lat{0.0f};
```